library/func_misc.py

```diff
@@ -228,7 +228,7 @@ def Call(self, args):
 
         s = val_ops.ToStr(
             val, 'maybe() expected Str, but got %s' % value_str(val.tag()),
-            loc.Missing)
+            args.BlameLoc())
```

Comment:
Does it make sense to pass the `(` token directly here?  I am not sure if this allocates a new location

Reply:
Ah. `BlameLoc()` just returns the left paren token and shouldn't allocate. I'll rename it for clarity, like you mentioned in the other comment, though